v2/tests/logger_perf/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+add_executable(logger_perf main.c timer.h timer_win32.c)
+target_link_libraries(logger_perf c_logging c_logging_v2)
+add_test(NAME logger_perf COMMAND logger_perf)
```

Comment:
>logger_perf [](http://example.com/codeflow?start=34&length=11)

I think for RFC level this is as good as it gets.

There's 2 minor outstanding issues:
1) logging examples seem to be old
2) whether there can be structured hierachically context (and therefore events)

But this is as good as it gets, seeing as the ETW has been reversed engineerd and disassembled :) 

Big cheers!

Will sign off.

Reply:
Yay, i think I have addressed everything at RFC level.
Shall leave this PR for reference open until we are done with all the tasks, but will file PRs one by one for .md, headers, etc. separately.